<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>magasin</class>
 <widget class="QWidget" name="magasin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>rechercher offre</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_recherche">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rechercher">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>rechercher</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>55</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Nom offre</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ID</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Prix</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>191</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Trier</string>
   </property>
   <widget class="QPushButton" name="pushButton_trier">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Trier </string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Nom offre</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ID</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Prix</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>551</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Decouvrir les offrs du notre magasin:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>420</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Afficher les offres</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stat">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>460</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Statistiques</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>380</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PDF</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
